lib/graphql_rails/model/input.rb

```diff
@@ -20,34 +23,20 @@ def graphql_input_type
         )
       end
 
-      def attribute(attribute_name, type: nil, enum: nil, **attribute_options)
```

Comment:
moved to https://github.com/samesystem/graphql_rails/pull/211/files#diff-eeb4f2e1d298fe120743ae3e31ca58dfbdd2e9579150dba0941ca1bfbb510b0cR39